third_party/move/move-model/tests/testsuite.rs

```diff
@@ -3,22 +3,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use codespan_reporting::{diagnostic::Severity, term::termcolor::Buffer};
-use move_binary_format::{
-    access::ModuleAccess,
-    file_format::{FunctionDefinitionIndex, StructDefinitionIndex},
-};
 use move_command_line_common::testing::EXP_EXT;
 use move_compiler::shared::PackagePaths;
-use move_model::{
-    options::ModelBuilderOptions, run_bytecode_model_builder, run_model_builder_with_options,
-};
+use move_model::{options::ModelBuilderOptions, run_model_builder_with_options};
 use move_prover_test_utils::baseline_test::verify_or_update_baseline;
 use std::path::Path;
 
 fn test_runner(
     path: &Path,
     options: ModelBuilderOptions,
-    check_from_bytecode: bool,
+    _check_from_bytecode: bool,
```

Comment:
why was this removed and what are the implications?

Reply:
This is dead code (only used in tests), and it would have been rather difficult to keep it working.